conf/messages

```diff
@@ -1093,3 +1093,14 @@ itemFiscalMarksChoice.heading = Does the {0} have any fiscal marks?
 itemFiscalMarksChoice.checkYourAnswersLabel = Fiscal marks
 itemFiscalMarksChoice.error.required = Select yes if the {0} has fiscal marks
 itemFiscalMarksChoice.change.hidden = fiscal marks
+
+itemGeographicalIndication.title = What is the geographical indication for the {0}?
+itemGeographicalIndication.heading = What is the geographical indication for the {0}?
+itemGeographicalIndication.hint.PDO = Enter the name of the Protected Designation of Origin (PDO) for the product. Include a register number if you have one.
+itemGeographicalIndication.hint.PGI = Enter the name of the Protected Geographical Indication (PGI) for the product. Include a register number if you have one.
+itemGeographicalIndication.hint.GI = Enter the name of the Geographical Indication (GI) for the product. Include a register number if you have one.
+itemGeographicalIndication.error.required = Enter details of the geographical indication
+itemGeographicalIndication.error.length = Geographical indication must be 350 characters or less
+itemGeographicalIndication.error.xss = Geographical indication must not contain < and > and : and ;
+itemGeographicalIndication.checkYourAnswersLabel = Designation of origin details
+itemGeographicalIndication.change.hidden = protected designation
```

Comment:
Should this be?
```suggestion
itemGeographicalIndication.change.hidden = designation of origin details
```
If the hidden change link content is different from the key then it could be confusing for users

Reply:
This is what is in the prototype.